docs/use_cases/agents.md

```diff
@@ -10,6 +10,21 @@ that query in order to return the correct result. The key agent components can i
 
 Research developments in LLMs (e.g. [ChatGPT Plugins](https://openai.com/blog/chatgpt-plugins)), LLM research ([ReAct](https://arxiv.org/abs/2210.03629), [Toolformer](https://arxiv.org/abs/2302.04761)) and LLM tooling ([LangChain](https://python.langchain.com/en/latest/modules/agents.html), [Semantic Kernel](https://github.com/microsoft/semantic-kernel)) have popularized the concept of agents.
 
+
+## LlamaIndex Agents
```

Comment:
i'd probably put this under the Agents + LlamaIndex section (below), and add as an additional bullet point (if you use OpenAI function API, we offer a lightweight wrapper for you to easily build agents yourself!) 

then subheading is `### Native OpenAIAgent`

Reply:
Moved